sources/Makefile

```diff
@@ -57,6 +66,10 @@ print-status:
 	@echo " | POSTGRES_DB: $(POSTGRES_DB) "
 	@echo " | POSTGRES_USER: $(POSTGRES_USER) "
 	@echo " | POSTGRES_PASSWORD: $(POSTGRES_PASSWORD) "
+	@echo " | Queue Configs"
+	@echo " | REDIS_PORT: $(REDIS_PORT) "
+	@echo " | REDIS_PASSWORD: $(REDIS_PASSWORD) "
```

Comment:
Even not being the Openshift password, I would prefer to remove them all from this print.